src/Uno.UI/UI/Xaml/ResourceResolver.cs

```diff
@@ -112,6 +113,60 @@ public static object ResolveResourceStatic(object key, Type type, object context
 		public static bool ResolveResourceStatic(object key, out object value, object context = null)
 			=> TryStaticRetrieval(new SpecializedResourceDictionary.ResourceKey(key), context, out value);
 
+		/// <summary>
+		/// Sets the default symbols font, assuming that the GlobalStaticResources have been initialized.
+		/// </summary>
+		/// <remarks>
+		/// This method is needed to search for the resource dictionary containing the appropriate resources
+		/// as the Application.Resources dictionary may change in structure based on how many resource dictionaries
+		/// are included.
+		/// </remarks>
+		internal static void SetSymbolsFontFamily()
+		{
+			// We assume here that we only have one dictionary with theme resources for performance reasons
+			if (FindSymbolFontFamilyDictionary() is { } genericDictionary)
+			{
+				var symbolsFontFamily = new FontFamily(global::Uno.UI.FeatureConfiguration.Font.SymbolsFont);
+
+				var themes = new[] { "Default", "Light", "HighContrast" };
+
+				foreach(var theme in themes)
```

Comment:
```suggestion
				foreach (var theme in themes)
```

Reply:
The sad part about this is that it's code generated by snippets, which are not reformatted in inclusion...

Reply:
@jeromelaban This is not supposed to happen. There is actually a code-style option for it:

https://github.com/unoplatform/uno/blob/f4d3c38b0a264e78801b08095107b84f7472df64/.editorconfig#L94

Do you know if this always has been the behavior for you or if it's a recent Roslyn regression?

Reply:
In case the `.editorconfig` is ignored due to some bug in Roslyn, does setting the option from VS UI help?

![image](https://user-images.githubusercontent.com/31348972/214341597-59f5253e-9000-4676-9d22-88e192e93db4.png)

Reply:
I'd think that it's always been that way, but did not take much attention :) That particular option is already set in my config, so it seems snippets don't follow that config indeed.

Reply:
I'll check in few days if I can reproduce and report to dotnet/roslyn.